docs/source/user-guide/cli.md

```diff
@@ -19,8 +19,7 @@
 
 # DataFusion Command-line Interface
 
-The DataFusion CLI allows SQL queries to be executed by an in-process DataFusion context, or by a distributed
-Ballista context.
+The DataFusion CLI allows SQL queries to be executed by an in-process DataFusion context.
 
 ```
 USAGE:
```

Comment:
According to https://github.com/apache/arrow-datafusion/pull/2604/ I think this entire block needs to be updated.

Reply:
i think it still need diff between `FLAGS ` and `OPTIONS `